datahub-web-react/src/app/ingestV2/source/builder/RecipeForm/constants.ts

```diff
@@ -608,6 +664,90 @@ export const RECIPE_FIELDS: RecipeFields = {
         ],
         advancedFields: [INGEST_STORIES, INGEST_APPLICATIONS, STATEFUL_INGESTION_ENABLED],
     },
+    [GLUE]: {
+        fields: [
+            GLUE_AWS_REGION,
+            GLUE_AWS_AUTHORIZATION_METHOD,
+            GLUE_AWS_ACCESS_KEY_ID,
+            GLUE_AWS_SECRET_ACCESS_KEY,
+            GLUE_AWS_SESSION_TOKEN,
+            GLUE_AWS_ROLE,
+            GLUE_CATALOG_ID,
+        ],
+        filterFields: [DATABASE_ALLOW, DATABASE_DENY, TABLE_ALLOW, TABLE_DENY],
+        advancedFields: [
+            GLUE_PLATFORM_INSTANCE,
+            ENV,
+            GLUE_EXTRACT_OWNERS,
+            GLUE_EXTRACT_TRANSFORMS,
+            GLUE_EMIT_S3_LINEAGE,
+            GLUE_INCLUDE_COLUMN_LINEAGE,
+            GLUE_PROFILING_ENABLED,
+            PROFILING_TABLE_LEVEL_ONLY,
+            GLUE_REMOVE_STALE_METADATA_ENABLED,
+        ],
+        hasDynamicFields: true,
+    },
+    [ORACLE]: {
+        fields: [
+            ORACLE_HOST_PORT,
+            ORACLE_USERNAME,
+            ORACLE_PASSWORD,
+            ORACLE_IDENTIFIER,
+            ORACLE_DATABASE,
+            ORACLE_SERVICE_NAME,
+        ],
+        filterFields: [SCHEMA_ALLOW, SCHEMA_DENY, TABLE_ALLOW, TABLE_DENY, VIEW_ALLOW, VIEW_DENY],
+        advancedFields: [
+            ORACLE_PLATFORM_INSTANCE,
+            ENV,
+            ORACLE_INCLUDE_TABLES,
+            ORACLE_INCLUDE_VIEWS,
+            ORACLE_INCLUDE_VIEW_LINEAGE,
+            ORACLE_INCLUDE_VIEW_COLUMN_LINEAGE,
+            ORACLE_PROFILING_ENABLED,
+            PROFILING_TABLE_LEVEL_ONLY,
+            ORACLE_EXTRACT_USAGE_HISTORY,
+            ORACLE_CONVERT_URNS_TO_LOWERCASE,
+            REMOVE_STALE_METADATA_ENABLED,
+        ],
+        hasDynamicFields: true,
+    },
+    [DREMIO]: {
+        fields: [
+            DREMIO_IS_DREMIO_CLOUD,
+            DREMIO_DREMIO_CLOUD_REGION,
+            DREMIO_DREMIO_CLOUD_PROJECT_ID,
+            DREMIO_HOSTNAME,
+            DREMIO_PORT,
+            DREMIO_TLS,
+            DREMIO_AUTHENTICATION_METHOD,
+            DREMIO_USERNAME,
+            DREMIO_PASSWORD,
+        ],
+        filterFields: [
+            SCHEMA_ALLOW,
+            SCHEMA_DENY,
+            DATASET_ALLOW,
+            DATASET_DENY,
+            DREMIO_PROFILE_ALLOW,
+            DREMIO_PROFILE_DENY,
+        ],
+        advancedFields: [
+            DREMIO_PLATFORM_INSTANCE,
+            ENV,
+            DREMIO_INGEST_OWNER,
+            DREMIO_INCLUDE_QUERY_LINEAGE,
+            PROFILING_ENABLED,
+            PROFILING_TABLE_LEVEL_ONLY,
+            REMOVE_STALE_METADATA_ENABLED,
+        ],
+        hasDynamicFields: true,
+    },
 };
 
-export const CONNECTORS_WITH_FORM = new Set(Object.keys(RECIPE_FIELDS));
+const ALL_CONNECTORS_WITH_FORM = Object.keys(RECIPE_FIELDS);
+export const CONNECTORS_WITH_FORM_INCLUDING_DYNAMIC_FIELDS = new Set(ALL_CONNECTORS_WITH_FORM);
+export const CONNECTORS_WITH_FORM = new Set(
```

Comment:
could update the name here to say `CONNECTORS_WITH_FORM_NO_DYNAMIC_FIELDS` otherwise i would assume this is all of the connectors with a form

Reply:
just pushed this up